packages/react-core/src/demos/ProgressStepperDemo.md

```diff
@@ -45,7 +45,7 @@ ProgressStepperDemo = () => {
         <br />
       </StackItem>
       <StackItem>
-        <div className="pf-v5-screen-reader" aria-live="polite">
+        <div className="pf-v6-screen-reader" aria-live="polite">
```

Comment:
There is an object defined for this in the `@patternfly/react-styles/css/utilities/Accessibility`
If you don't update it in your PR, that is ok because I have an [issue](https://github.com/patternfly/patternfly-react/issues/10955) open to update this throughout the code base.